services/monitoring/grafana/terraform/dashboards.tf

```diff
@@ -0,0 +1,35 @@
+// Import subfolders using an external script
+data "external" "subfolders" {
+  program = ["bash", "${path.module}/../../../../scripts/tf_helper_list_subfolders.sh", "${path.module}/../assets/shared/dashboards"]
+}
+
+// Local mappings of folder names to their paths
+locals {
+  folder_map = data.external.subfolders.result
+}
+
+// Create Grafana folders for each subfolder
+resource "grafana_folder" "subfolders" {
+  for_each = local.folder_map
+
+  title = each.key // Use each.key to access each folder's name
+}
+
+// Function to list all JSON files within a directory
+data "external" "dashboard_files" {
+  for_each = local.folder_map
+
+  program = ["bash", "${path.module}/../../../../scripts/tf_helper_list_json_files_in_folder.sh", "${path.module}/../assets/shared/dashboards/${each.key}"]
```

Comment:
I wonder if we can keep this scripts inside terraform folder to avoid having `../../../../`